data/mock-7/dataset-metadata.tsv

```diff
@@ -1,7 +1,7 @@
 name	value
 citation	doi: 10.1016/j.cell.2012.10.052
 qiita-id	NA
-raw-data-url	http://metagenomics.anl.gov/metagenomics.cgi?page=MetagenomeProject&project=1319
+raw-data-url	ftp://ftp.microbio.me/pub/illumina-mock-communities-raw-data/Turnbaugh3/
```

Comment:
@gregcaporaso In the microbio.me FTP, we had the original dataset names for all directories and never changed them. Turnbaugh-3 would be consistent with this. If we are moving the raw data elsewhere, we can rename the directories there... until we decide on the final home I will leave this as is. Does that sound ok?